pkg/kubectl/cmd/util/helpers.go

```diff
@@ -203,31 +205,34 @@ func StandardErrorMessage(err error) (string, bool) {
 	return "", false
 }
 
-// MultilineError returns a string representing an error that splits sub errors into their own
-// lines. The returned string will end with a newline.
-func MultilineError(prefix string, err error) string {
-	if agg, ok := err.(utilerrors.Aggregate); ok {
+// multilineErrorString returns a string representing an error that splits sub errors into their own
+// lines. The returned string will end with a newline. The prefix should not end in . or : or \n.
+func multilineErrorString(err string, subErr error) string {
+	if subErr == nil {
+		return fmt.Sprintf("%s.\n", err)
+	}
+	if agg, ok := subErr.(utilerrors.Aggregate); ok {
 		errs := utilerrors.Flatten(agg).Errors()
 		buf := &bytes.Buffer{}
 		switch len(errs) {
 		case 0:
-			return fmt.Sprintf("%s%v\n", prefix, err)
+			return fmt.Sprintf("%s: %v\n", err, subErr)
 		case 1:
-			return fmt.Sprintf("%s%v\n", prefix, messageForError(errs[0]))
+			return fmt.Sprintf("%s: %v\n", err, messageForError(errs[0]))
 		default:
-			fmt.Fprintln(buf, prefix)
+			fmt.Fprintf(buf, "%s:\n\n", err)
 			for _, err := range errs {
 				fmt.Fprintf(buf, "* %v\n", messageForError(err))
 			}
 			return buf.String()
 		}
 	}
-	return fmt.Sprintf("%s%s\n", prefix, err)
+	return fmt.Sprintf("%s: %s\n", err, subErr)
 }
 
-// MultipleErrors returns a newline delimited string containing
+// multipleErrors returns a newline delimited string containing
 // the prefix and referenced errors in standard form.
-func MultipleErrors(prefix string, errs []error) string {
+func multipleErrors(prefix string, errs []error) string {
```

Comment:
This in fact is used downstream. Having invisible downstream usage of those auxiliary functions really hurts.